# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/moz.build
# Commit: 789ce1a19e9f
# Full Hash: 789ce1a19e9f09a806dde2a27480963f45832218
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-06-10 09:49:36
# Regressor Bug: 1564235
# File Overlap Count: 3
# Description:
#   Bug 1564235 - P2 Create NavigationPreloadManager.webidl and implement webidl binding. r=dom-worker-reviewers,webidl,asuth
#   
#   This patch creates NavigationPreloadManager.webidl and implements the webidl binding.
#   
#   Depends on D116913
# ==============================================================================

diff -r a4cd096b0ba7 -r 789ce1a19e9f dom/serviceworkers/moz.build
--- a/dom/serviceworkers/moz.build	Thu Jun 10 00:48:10 2021 +0000
+++ b/dom/serviceworkers/moz.build	Thu Jun 10 00:48:11 2021 +0000
@@ -13,6 +13,7 @@
     "FetchEventOpParent.h",
     "FetchEventOpProxyChild.h",
     "FetchEventOpProxyParent.h",
+    "NavigationPreloadManager.h",
     "ServiceWorker.h",
     "ServiceWorkerActors.h",
     "ServiceWorkerChild.h",
@@ -46,6 +47,7 @@
     "FetchEventOpParent.cpp",
     "FetchEventOpProxyChild.cpp",
     "FetchEventOpProxyParent.cpp",
+    "NavigationPreloadManager.cpp",
     "RemoteServiceWorkerContainerImpl.cpp",
     "RemoteServiceWorkerImpl.cpp",
     "RemoteServiceWorkerRegistrationImpl.cpp",